__tests__/pages/dashboard.test.js

```diff
@@ -14,7 +16,24 @@ expect.extend(toHaveNoViolations)
 jest.mock('next-auth/react')
 enableFetchMocks()
 
+jest.mock('cookies-next', () => ({
+  getCookie: () => 'default',
```

Comment:
This doesn't look to be doing quite what we expect; in the coverage report it's showing that this line (https://dts-stn.github.io/sc-digital-centre/refs/pull/409/merge/coverage/lcov-report/pages/dashboard.js.html#L275) isn't getting hit; which it should if getCookie returns `default`. Line 55 has an example of how a return value of getSession is being mocked.

Reply:
nvm, this is working; coverage wan't updated when I looked at it.